src/docs/010_manual/20_install.adoc

```diff
@@ -8,10 +8,24 @@
 
 include::../_feedback.adoc[]
 
-== Get the Wrapper
+== Getting the Wrapper
 
-The basic strategy on how to install docToolchain has changed with V2.x.
-You now install a small wrapper scripts to your project which does the rest for you:
+There have been a few changes to the way docToolchain can be installed since its V2.x release. We built a simple wrapper script to takes care of the everything for you.
```

Comment:
The way docToolchain is installed has changed since the v2.x release. It now uses a simple wrapper script which takes care of everything.

**@Bennykillua - I like your approach here. My suggestion above makes it even shorter and more to the point. Plus grammar fixes.**